graphql/resolver.go

```diff
@@ -96,6 +96,10 @@ func New(apiURL string) Config {
 		if !isStringMap {
 			return nil, ResourceNotFound.Send(ctx, "Project not specified")
 		}
+
+		if hasOperationContext := graphql.HasOperationContext(ctx); !hasOperationContext {
```

Comment:
This might not actually be enough because HasOperationContext checks if the value exists but it doesn't check that the value is nil (which would still panic) but this would still narrow it down. You could alternative add a panic handler 
https://github.com/evergreen-ci/evergreen/blob/f2381fe506626079db16089a11ab98bb7f690181/model/task/task.go#L1660

Reply:
I think it might be okay! I tried testing it manually and it seemed fine, and in the corresponding unit tests I wrote I never defined the operation context but the tests didn't panic

Reply:
Could we remove this check? If the error was originating from apollo tracing this presumably is no longer needed. If we do still run into it its probably some other error.

Reply:
No, this will still occur if the request is malformed, because the operation context is not created

the only reason this was not reached was because the ApolloTracer was declared before the SplunkTracer

but if it's not important we can remove it